recipe/meta.yaml

```diff
@@ -12,20 +12,24 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 2
+  number: 3
   skip: true  # [win]
   skip: true  # [py2k]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - python                                # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}    # [build_platform != target_platform]
+    - mysql                                 # [build_platform != target_platform]
   host:
     - python
     - wheel
     - setuptools
     - setuptools_scm 6.0.1
     - setuptools_scm_git_archive
     - libtiledb-sql >=0.10
+    - mysql                                # [build_platform != target_platform]
```

Comment:
Typically `# [build_platform != target_platform]` is only need for build requirements. Why would `mysql` need to be explicitly installed into the host environment only when cross-compiling?

Reply:
Because I only get a ```mysql.h not found``` error when cross compiling for arm macOS. The error is resolved by adding mysql to the dependencies. I have not tried to remove the dependency from the host section yet though.

Reply:
The arm build is successful now. I have another issue unrelated to this PR. See logs

Reply:
> I have not tried to remove the dependency from the host section yet though.

Seems worth a try to see if it still cross-compiles without the host requirement